docs/nginx.md

```diff
@@ -182,7 +182,7 @@ Now, start your stack:
 
 Login with a "push" authorized user (using `testuser` and `testpassword`), then tag and push your first image:
 
-    docker login -p=testuser -u=testpassword -e=root@example.ch myregistrydomain.com:5043
+    docker login -u=testpassword -p=testuser -e=root@example.ch myregistrydomain.com:5043
```

Comment:
looks like the values are still swapped; should be

    -u=testuser -p=testpassword